### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by Paasdag** [community/community#110820] [GitHub Education]
### Hi everyone,

Hi everyone so basically my school does not do school i'ds and i dont have any of the other information since my parents got that, am i still eligible for github student developer pack?

**Comment by SebastienXia** [comment]
Hi, @Paasdag 
Everything you need to be eligible for the **Student developer pack** and below.

![Capture](https://github.com/community/community/assets/105004912/c6636024-c61c-4647-abe5-cd57262364ce)

If you do not meet these conditions you will unfortunately not be eligible. 

For more details with this, you can check the *documentation* below: []()https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student
